<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceQCM</class>
 <widget class="QDialog" name="InterfaceQCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuestionSuivante">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEffacerReponse">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuestionPrecedente">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
